### PR TITLE
Update error.go 

### PR DESCRIPTION
**Pull request by mervegonc**
#51  implement ErrorDlrNotFound

### CODE REVIEW

File: internal/domain/model/object/error.go [hunk 1]
```diff
@@ -13,13 +13,20 @@ var ERRORS []error = []error{
...
 var (
 	ErrorNone         error = nil
 	ErrorUserNotFound error = errors.New(smodel.ErrBase + smodel.ErrSep + ErrUser + smodel.ErrSep + smodel.ErrNotFound)
+	ErrorDlrNotFound  error = errors.New(smodel.ErrBase + smodel.ErrSep + ErrDlr + smodel.ErrSep + smodel.ErrNotFound)
 )
 
+// GetErrors returns the list of all errors
 func GetErrors() []error {
-	return ERRORS
+	return []error{
```
Comment (husamettinarabaci) [inline internal/domain/model/object/error.go:29]: You have to add all error in the 
var ERRORS []error
and return that.

**Review by husamettinarabaci — CHANGES_REQUESTED** [review]

**Review by Sddilora — CHANGES_REQUESTED** [review]
You will also add ErrorDlrNotFound to 
```go
var ERRORS []error = []error{
	ErrorNone,
	ErrorUserNotFound,
}
```
 here

**Comment by husamettinarabaci** [comment]
> You will also add ErrorDlrNotFound to 
> 
> ```go
> 
> var ERRORS []error = []error{
> 
> 	ErrorNone,
> 
> 	ErrorUserNotFound,
> 
> }
> 
> ```
> 
>  here

> And you have to returnt ERRORS

File: internal/domain/model/object/error.go [hunk 2]
```diff
@@ -9,17 +9,27 @@ import (
...
+		ErrorNone,
+		ErrorUserNotFound,
+		ErrorDlrNotFound,
+		
+	}
```
Comment (Sddilora) [inline internal/domain/model/object/error.go:34]: You need to return ERRORS :)

**Review by Sddilora — CHANGES_REQUESTED** [review]

**Pull request closed without merging**